andela/mazus-ah-frontend
# #168323925 Changed flash messages from toaster to antd

### What does this PR do?
Removes the Previous toaster message and uses antd

#### Description of Task to be completed?
adds antd to authactions and password reset actions and created an scss file for the styling

#### How should this be manually tested?

> git clone branch
> npm run dev to start the server 
> navigate to the sign up or signin page and submit
> you should get a message with the new antd message

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#168323925](https://www.pivotaltracker.com/story/show/168323925)

#### Screenshots (if appropriate)
